dcbr-web/src/store/registration/actions.ts

```diff
@@ -39,7 +39,7 @@ export const actions: ActionTree<RegistrationState, RootState> = {
   postalCode(context: any, payload: string) {
     context.commit("postalCode", payload);
   },
-  homeAsOperation(context: any, payload: boolean) {
+  homeAsOperation(context: any, payload: string) {
```

Comment:
Why are we changing these to `string` from `boolean`? Based on the action(s) it seems like `boolean` would be the right type to use, just wondering what made us decide to go with an empty string for an unassigned value instead (applies to next files as well).

Reply:
Using booleans for the true/false radio buttons required us to set false as the default value, which had the form already checked to "No" for all the Yes/No questions. Using strings we can set this default value to a blank string, the back end interprets "true" as true and "false" as false without modification.

Reply:
Oh, gotcha. We want to use an unset default state. Could we use booleans and set it to `undefined` instead?

Reply:
@sam-warren let me know if you get to try the above and see if it is an option. It is not likely a HUGE issue in this case, but keeping data types consistent with the context they are being used in is generally a good idea.